Src/Sarif.PatternMatcher/ValidatingVisitor.cs

```diff
@@ -37,12 +39,14 @@ public override Result VisitResult(Result node)
                 // Our validation messages currently look like so.
                 // {0:scanTarget}' contains {1:validationPrefix}{2:encoding}{3:secretKind}{4:validationSuffix}{5:validatorMessage}
                 string message = null;
+                IDictionary<string, string> options = new Dictionary<string, string>();
```

Comment:
>IDictionary<string, string> [](start = 16, length = 27)

use 'var'

Reply:
the method below needs IDictionary, so we need to enforce using the interface